### DISCUSSION TITLE
[WASM] ... no longer builds using latest DEV bits

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#12918] [Q&A]
### Current behavior

I'm not sure if caused by #12755 or not, but upgrading to latest DEV bits (see screenshot below) prevents my solution from building throwing these errors
```
9>C:\tmp\unusual-src-dev\Unusual.Wasm\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGenerator\MaterialTheme_84d781f597680f94b23c291ff1c40a24.cs(1190,14,1190,108): error CS1029: #error: 'Property for DropDownPreferredPlacement cannot be found on Microsoft.UI.Xaml.Controls.ComboBox'
9>C:\tmp\unusual-src-dev\Unusual.Wasm\Uno.UI.SourceGenerators\Uno.UI.SourceGenerators.XamlGenerator.XamlCodeGenerator\MaterialTheme_84d781f597680f94b23c291ff1c40a24.cs(6125,15,6125,119): error CS1029: #error: 'Property for ShouldFallBackToNativeScrollBars cannot be found on Microsoft.UI.Xaml.Controls.ScrollViewer'
```
@TopProgrammer77 @dr1rrb FYI

**Update**: 
After further investigation I found that the culprit are these lines in my `MaterialTheme.xaml `style file:
```
<not_win:Setter Property="(uno:ScrollViewer.ShouldFallBackToNativeScrollBars)" Value="False" />
<xamarin:Setter Property="uno:ComboBox.DropDownPreferredPlacement" Value="Below" />												
```
That file is a copy&paste from UNO Material theme sources ~1.5 years back + some heavy customization I applied meanwhile. However, the 2 lines above had been part of the original UNO Material theme source.

I needed to know if I could savely remove those lines of if should replace them by something different. Note: I'm only using the WindowsAppSDK and the WASM head at this time.

<img width="1280" alt="image" src="https://github.com/unoplatform/uno/assets/26179242/e086115f-8145-43f6-9303-8c8216cdb573">

**Comment by jeromelaban** [comment]
Thanks for the report. 

The master of Uno is now updated to the upcoming Uno 5 release, which contains some breaking changes that are currently being [documented here](https://github.com/unoplatform/uno/blob/master/doc/articles/migrating-from-previous-releases.md). Libraries such as uno.material have been updated as well to handle those changes.

**Reply by DierkDroth** [reply]
Thanks @jeromelaban 

I checked the link you provided and could not find any reference for neither `ShouldFallBackToNativeScrollBars ` nor `DropDownPreferredPlacement`. What am I missing?

**Reply by jeromelaban** [reply]
The default styles present in Uno.Themes are working properly with those lines and should be working properly with your app as well. We do not have a modification suggestion at this time.

**Reply by DierkDroth** [reply]
@jeromelaban I'm confused. Are you saying the current Uno.Themes sources would hold those lines and still compile/build using the 'latest' bits which I tried to apply?

If so, what should I then be looking at in order to make my project compilable again?

**Reply by jeromelaban** [reply]
Yes, the latest uno.theme dev bits have those two lines and builds properly. As I mentioned, if your project does not build, we do not have an upgrade path yet. You can open an issue with a repro sample that shows what's not building.

**Reply by DierkDroth** [reply]
I did some digging and still don't know what would cause that.
- as for `DropDownPreferredPlacement `at least I needed to add a new package (`UNO.WinUI`) which was not needed in my solution until now ... I don't want to do that for just fixing a detail which what was working so far
- as for `ShouldFallBackToNativeScrollBars `I would not know where to begin with

I just removed both lines from my `MaterialThemes.xaml`. Appears this has no negative impact on ComboBox nor CalenderView (where they had been consumed).
